recipes/khiops-core/conda_build_config.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
This should come from the global pinning. I guess it is best to delete the whole file.